data/json/items/gun/8x40mm.json

```diff
@@ -122,8 +122,7 @@
     "reload": 400,
     "barrel_length": 6,
     "valid_mod_locations": [ [ "barrel", 1 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "rail", 1 ], [ "sights", 1 ], [ "stock", 1 ] ],
-    "magazine_well": 6,
```

Comment:
Why remove this?

Reply:
To increase the size of the RM298.

Reply:
And why does it need to be larger?

Reply:
I'm going to remove the mounted flag and re-add the magazine_well